src/components/Home/WhatsNewModal/index.tsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React, { useState } from 'react'
+import cn from 'classnames'
+import FocusLock from 'react-focus-lock'
+
+import Video from '../UseCases/Video'
+import Link from '../../Link'
+
+import { ReactComponent as CloseSvg } from '../../../../static/img/close-icon.svg'
+
+import * as styles from './styles.module.css'
+
+const version = '2.8.3'
+
+const WhatsNewModal: React.FC = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false)
+  let pageCloseEventListener: () => void = () => null
+  let keyupCloseEventListener: () => void = () => null
+
+  const closeModal = (): void => {
+    setIsModalOpen(false)
+    pageCloseEventListener()
+    keyupCloseEventListener()
+    document.body.style.overflow = 'visible'
+  }
+
+  const handleContainerClick = (event: MouseEvent): void => {
+    const el = event.target as HTMLDivElement
+
+    if (el.classList.contains(styles.container)) {
+      closeModal()
+    }
+  }
+
+  const handlePageKeyup = (event: KeyboardEvent): void => {
+    if (event.key === 'Escape') {
+      closeModal()
+    }
+  }
+
+  const setupModalEventListeners = (): void => {
+    document.addEventListener('click', handleContainerClick)
+    document.addEventListener('keyup', handlePageKeyup)
+
+    pageCloseEventListener = (): void =>
+      document.removeEventListener('click', handleContainerClick)
+    keyupCloseEventListener = (): void =>
+      document.removeEventListener('keyup', handlePageKeyup)
+    document.body.style.overflow = 'hidden'
+  }
+
+  const handleButtonClick = (): void => {
+    if (isModalOpen) {
+      closeModal()
+    } else {
+      setupModalEventListeners()
+      setIsModalOpen(true)
+    }
+  }
+
+  return (
+    <FocusLock disabled={!isModalOpen}>
+      <div
+        className={cn(styles.container, isModalOpen && styles.containerOpen)}
+      >
+        <div
+          aria-label={`What's new in ${version}`}
+          tabIndex={0}
+          role="dialog"
+          aria-modal="true"
+        >
+          <button
+            onClick={handleButtonClick}
+            className={cn(styles.button, isModalOpen && styles.buttonExit)}
+          >
+            {!isModalOpen && `What's new in ${version}`}
+            {isModalOpen && <CloseSvg width={20} height={20} />}
+          </button>
+          <div className={styles.modal}>
+            <h2 className={styles.title}>
+              LiveTask, experiment name check, try reflink and more in DVC 2.8.3
```

Comment:
Does the title or paragraph text need revising?

Reply:
Do we have a link yet? I can add the link if we do!

Reply:
Okay, we've decided on https://dvc.org/blog/ml-experiment-versioning. Is there a way to make the link more prominent or make the title clickable so it's more obvious to click on something? cc @yalozhkin

Reply:
Also, let's slightly update the text.

Title: ML Experiment Versioning

Paragraph: Versioning experiments combines the benefits of version control and experiment tracking.

Reply:
> Okay, we've decided on https://dvc.org/blog/ml-experiment-versioning. Is there a way to make the link more prominent or make the title clickable so it's more obvious to click on something? cc @yalozhkin

Good idea! I'll turn the title into a link.